modules/products/ui/components/products-filters.tsx

```
@@ -1,11 +1,12 @@
 "use client";
 
 import { useState } from "react";
-import { ChevronDownIcon, ChevronRightIcon } from "lucide-react";
+import { ChevronDownIcon, ChevronRightIcon, Key } from "lucide-react";
```

ESLint

[error] 'Key' is defined but never used.

(@typescript-eslint/no-unused-vars)